modules/Attendance/attendance_take_byPerson_editProcess.php

```diff
@@ -23,9 +23,9 @@
 //Module includes
 include './moduleFunctions.php';
 
-$gibbonAttendanceLogPersonID = isset($_POST['gibbonAttendanceLogPersonID'])? $_POST['gibbonAttendanceLogPersonID'] : '';
-$gibbonPersonID = isset($_POST['gibbonPersonID'])? $_POST['gibbonPersonID'] : '';
-$currentDate = isset($_POST['currentDate'])? $_POST['currentDate'] : dateConvertBack($guid, date('Y-m-d'));
+$gibbonAttendanceLogPersonID = $_POST['gibbonAttendanceLogPersonID'] ?? '';
+$gibbonPersonID = $_POST['gibbonPersonID'] ?? '';
+$currentDate = $_POST['currentDate'] ?? dateConvertBack($guid, date('Y-m-d'));
```

Comment:
Another spot. See if you can examine the syntax and come up with a way to write this as a null coalesce.

Reply:
Ross and I discussed this with you, and we decided that it's okay.